hknweb/events/templates/events/event_update.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}Edit {{ event.name }}{% endblock %}
+
+{% block header %}
+<link rel="stylesheet" href="{% static "events/style.css" %}">
+<style>
+    .heading {
+        display: none;
+    }
+    .content {
+        padding-top: 1em;
+        padding-bottom: 1em;
+    }
+    @media screen and (min-width: 1000px) {
+        .content {
+            padding-top: 1.5em;
+            padding-bottom: 1.5em;
+        }
+    }
+    .centered {
+        max-width: 700px;
+        margin: 0 auto;
+    }
+</style>
```

Comment:
Should we be putting these into base.html or base.css if they modify elements from base.html?

Reply:
But we don't want it to apply for all pages right?